### PR TITLE
feat:added modal for sharing information

### PR DESCRIPTION
**Pull request by KcPele**
## Description
created the modal for sharing information
_Concise description of proposed changes, We recommend using screenshots and videos for better description_
Screenshot from 2024-06-25 21-22-54

## Additional Information

- [ ] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [ ] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes #17 _

_Note: If your changes are small and straightforward, you may skip the creation of an issue beforehand and remove this section. However, for medium-to-large changes, it is recommended to have an open issue for discussion and approval prior to submitting a pull request._

Your ENS/address:

### CODE REVIEW

File: packages/nextjs/components/OnchainInfoModal.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+                <input id="selectedProject" type="checkbox" className=" w-[18px] h-[18px]  accent-blue-600" />
+                <span className="ml-4">Selected project</span>
+              </label>
+              <label className="inline-flex items-center" htmlFor="otherDimention">
+                <input id="otherDimention" type="checkbox" className=" w-[18px] h-[18px]  accent-blue-600" />
+                <span className="ml-4">Other dimension</span>
+              </label>
+            </div>
+            <div className="mt-4 flex items-center justify-center gap-6">
+              <div className="py-1 max-w-[72px] w-full px-4 border  border-neutral-400/30 rounded-xl">
```
Comment (Franpastoragusti) [inline packages/nextjs/components/OnchainInfoModal.tsx]: add cursor-ponter to the social buttons

**Comment by Franpastoragusti** [comment]
the rest is perfect, we will implement all the behavior later

File: packages/nextjs/components/OnchainInfoModal.tsx (hunk 1 above)
Comment (KcPele) [reply packages/nextjs/components/OnchainInfoModal.tsx]: done

**Review by Franpastoragusti — APPROVED** [review]

**Review by Franpastoragusti — APPROVED** [review]

File: packages/nextjs/app/page.tsx [hunk 2]
```diff
@@ -1,3 +1,6 @@
+"use client";
+
+import { useState } from "react";
```
Comment (escottalexander) [inline packages/nextjs/app/page.tsx]: I messed up when I resolved conflicts, Can you remove this "useState" declaration and then we can merge.

**Review by escottalexander — CHANGES_REQUESTED** [review]

File: packages/nextjs/app/page.tsx (hunk 2 above)
Comment (KcPele) [reply packages/nextjs/app/page.tsx]: Resolved

**Review by escottalexander — APPROVED** [review]
Nice work!

**Pull request merged**